spec/support/features/book_helper.rb

```diff
@@ -0,0 +1,8 @@
+module Features
+  def create_book(book_title, book_author)
+    click_on "Add a new book"
+    fill_in "Title", with: book_title
+    fill_in "Author", with: book_author
+    click_on "Create Book"
+  end 
+end
```

Comment:
I would add this as a method on the test that uses it. Since it's only used in one place that will make it easier to read and modify later. I would use something like this for helpers that are used in a lot of different feature tests